apps/src/gamelab/GameLabP5.js

```diff
@@ -564,13 +564,27 @@ GameLabP5.prototype.drawDebugSpriteColliders = function () {
 /**
  * Instantiate a new p5 and start execution
  */
-GameLabP5.prototype.startExecution = function () {
+GameLabP5.prototype.startExecution = function (gamelab) {
   new window.p5(function (p5obj) {
       this.p5 = p5obj;
       this.p5.useQuadTree(false);
       this.setP5FrameRate();
       this.gameLabWorld = new GameLabWorld(p5obj);
 
+      p5obj.showMobileControls = function (
```

Comment:
Do we get anything out of putting this on our p5 object?  It seems like this could live someplace else.

Reply:
This is the most convenient place for it to be marshaled in the global space. But I can look at wedging it in specifically in other spot.